Index: GConf-2.28.1/gconf/gconf-internals.c
===================================================================
--- GConf-2.28.1.orig/gconf/gconf-internals.c	2010-05-29 10:45:17.894434221 +0200
+++ GConf-2.28.1/gconf/gconf-internals.c	2010-05-29 10:47:17.758430078 +0200
@@ -2882,7 +2882,7 @@ gconf_activate_server (gboolean  start_i
     g_set_error (error,
                  GCONF_ERROR,
                  GCONF_ERROR_NO_SERVER,
-                 _("Failed to contact configuration server; some possible causes are that you need to enable TCP/IP networking for ORBit, or you have stale NFS locks due to a system crash. See http://projects.gnome.org/gconf/ for information. (Details - %s)"),
+                 _("Failed to contact configuration server; the most common cause is a missing or misconfigured D-Bus session bus daemon. See http://projects.gnome.org/gconf/ for information. (Details - %s)"),
                  failure_log->len > 0 ? failure_log->str : _("none"));
 
   g_string_free (failure_log, TRUE);
